src/routing/bootstrap.rs

```diff
@@ -209,35 +207,35 @@ impl<'a> State<'a> {
             }
         }
 
-        error!("{} Message sender unexpectedly closed", self.node);
```

Comment:
still keeping this log ?

Reply:
Don't think it's needed. There is now separate error variant for this and we log those errors already.